meta-mender-tegra/recipes-bsp/u-boot/patches/0013-Jetson-Nano-mender-environment-in-mmc.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/include/configs/p3450-porg.h b/include/configs/p3450-porg.h
+index 7be05f3..427fb60 100644
+--- a/include/configs/p3450-porg.h
++++ b/include/configs/p3450-porg.h
+@@ -31,8 +31,11 @@
+ #define BOOT_TARGET_DEVICES(func) \
+ 	func(MMC, mmc, 1)
+ 
+-/* Environment s/b at end of SPI, fix it later */
+-#define CONFIG_ENV_IS_NOWHERE
++/* Env is located in it's own partition */
++/* https://github.com/madisongh/meta-tegra/issues/188 */
++#define CONFIG_SYS_MMC_ENV_DEV		1
++#define CONFIG_ENV_IS_IN_MMC
++#define CONFIG_ENV_OFFSET		12582912
```

Comment:
@moqmar I wouldn't expect you'd need to hard code this, I think mender should provide it.  See https://github.com/Trellis-Logic/meta-mender-community/blob/b91ba8a1a85732d9000b6bf32506664e2b979d22/meta-mender-tegra/recipes-bsp/u-boot/patches/0012-Update-environment-defaults-for-tegra.patch for instance.  Defining it here would mean you'll need to change it whenever you change `MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET` I believe

Reply:
Resolved in the comment at https://github.com/Trellis-Logic/meta-mender-community/pull/9#discussion_r358136171